views/frontend/home.php

```diff
@@ -2,29 +2,28 @@
 <div class="row">
 
 <?php
-$posts = $model_article->get_posts_blog();
 foreach($posts as $post){
         ?><div class="col l6 m6 s12">
             <div class="card">
                 <div class="card-content">
-                    <h1 class="grey-text text-darken-2"><?= $post->title ?></h1>
-                    <h5 class="grey-text">Le <?= date("d/m/Y à H:i",strtotime($post->date)); ?> par <?= $post->name ?></h5>
+                    <h1 class="grey-text text-darken-2"><?= $post->getTitle() ?></h1>
+                    <h5 class="grey-text">Le <?= date("d/m/Y à H:i",strtotime($post->getDate())); ?> par <?= $post->getTitle() ?></h5>
                 </div>
                 <div class="row">
                     <div class="col s12">
-                        <?= substr(nl2br($post->chapo),0,1200) ?>...
+                        <?= substr(nl2br($post->getChapo()),0,1200) ?>...
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'substr'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)